scripts/kind-e2e/e2e.sh

```diff
@@ -39,7 +39,7 @@ function install_helm() {
             echo Installing helm on cluster${i}.
             kubectl -n kube-system create serviceaccount tiller
             kubectl create clusterrolebinding tiller --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
-            helm init --service-account tiller
+            helm init --service-account tiller > /dev/null 2>&1
```

Comment:
I know the output here is boring, but IME it’s better to keep the output because stuff ends up going wrong :-(.

Reply:
> I know the output here is boring, but IME it’s better to keep the output because stuff ends up going wrong :-(.

Yes i know what you mean, the outputs are way too long :), i have added echo to each function but you are correct we probably should keep the output.